src/pages/Home.js

```diff
@@ -57,18 +77,59 @@ const Home = () => {
                         params: p
                 })
                         .then(function (response) {
+
+                                let content_type;
```

Comment:
```suggestion
                                let contentType;
```
基本的にJSはキャメルケースで書きましょう